### ISSUE TITLE
Fervo case study

### ISSUE THREAD

**Issue by softwareengineerprogrammer** [NREL/GEOPHIRES-X#202]
Per https://github.com/NREL/GEOPHIRES-X/issues/87?title=Add+examples/case-studies+of+real-world+geothermal+systems

Papers:
1. https://eartharxiv.org/repository/view/5704/
2. Drilling cost paper https://pangea.stanford.edu/ERE/db/GeoConf/papers/SGW/2024/Elsadi.pdf?t=1706897516

**Comment by softwareengineerprogrammer** [comment]
Initial parameters & result merged in https://github.com/NREL/GEOPHIRES-X/pull/203; [`Fervo_Norbeck_Latimer_2024.txt`](https://github.com/NREL/GEOPHIRES-X/blob/main/tests/examples/Fervo_Norbeck_Latimer_2024.txt) and [`Fervo_Norbeck_Latimer_2024.out`](https://github.com/NREL/GEOPHIRES-X/blob/main/tests/examples/Fervo_Norbeck_Latimer_2024.out) are still considered work-in-progress until @kfbeckers and/or @malcolm-dsider further review & refine.

**Comment by softwareengineerprogrammer** [comment]
Issue was automatically closed by merge of https://github.com/NREL/GEOPHIRES-X/pull/204 - re-opened because PR was not fully reviewed by myself or by geothermal SME

**Comment by HuihuiYangshell** [comment]
Thanks a lot, they are really helpful.

**Comment by softwareengineerprogrammer** [comment]
Documenting @kfbeckers' current version of Project Cape Case Study for posterity:

Link: https://gtp.scientificwebservices.com/geophires?shared-geophires-result-id=c0bb86de-1356-4cab-9907-faf4005ecc3d

Input:
```
Reservoir Model, 1
Reservoir Volume Option, 1
Reservoir Density, 2800
Reservoir Depth, 8500 feet
Reservoir Heat Capacity, 1000
Reservoir Thermal Conductivity, 2.7
Number of Fractures, 100
Fracture Shape, 4
Fracture Height, 300
Fracture Width, 2400
Fracture Separation, 15
Number of Segments, 1
Gradient 1, 73
Plant Outlet Pressure, 1000 psi
Number of Injection Wells, 12
Number of Production Wells, 12
Well Geometry Configuration, 4
Has Nonvertical Section, True
Multilaterals Cased, True
Number of Multilateral Sections, 2
Total Nonvertical Length, 5000 feet
Production Flow Rate per Well, 98
Production Well Diameter, 8
Injection Well Diameter, 8
Well Separation, 365 feet
Injection Temperature, 60 degC
Injection Wellbore Temperature Gain, 3
Reservoir Impedance, 0.01
Ramey Production Wellbore Model, 1
Utilization Factor, .9
Water Loss Fraction, 0.02
Maximum Drawdown, 1
Ambient Temperature, 10 degC
Surface Temperature, 10 degC
End-Use Option, 1
Plant Lifetime, 15
Power Plant Type, 2
Circulation Pump Efficiency, 0.80
Exploration Capital Cost, 0
Well Drilling and Completion Capital Cost, 4.8
Reservoir Stimulation Capital Cost, 10
Economic Model, 3
Starting Electricity Sale Price, 0.15
Ending Electricity Sale Price, 1.00
Electricity Escalation Rate Per Year, 0.004053223
Electricity Escalation Start Year, 1
Fraction of Investment in Bonds, .5
Combined Income Tax Rate, .3
Gross Revenue Tax Rate, 0
Inflated Bond Interest Rate, .05
Inflated Equity Interest Rate, .08
Inflation Rate, .02
Investment Tax Credit Rate, 0
Inflation Rate During Construction, 0.05
Property Tax Rate, 0
Print Output to Console, 1
Time steps per year, 10
```

**Comment by softwareengineerprogrammer** [comment]
> Issue was automatically closed by merge of #204 - re-opened because PR was not fully reviewed by myself or additional geothermal SME

Resolved by https://github.com/NREL/GEOPHIRES-X/pull/220
